ArduPlane/quadplane.cpp

```diff
@@ -1457,7 +1457,12 @@ void QuadPlane::update(void)
     if (!setup()) {
         return;
     }
-    
+
+    if ((ahrs_view != NULL) && (_last_ahrs_trim_pitch - ahrs_trim_pitch.get()) > 0.1f) {
```

Comment:
(_last_ahrs_trim_pitch - ahrs_trim_pitch.get()) 
i think this should be abs(_last_ahrs_trim_pitch - ahrs_trim_pitch.get()) 
otherwise you can only reduce the trim

Reply:
thanks, that was a dumb mistake